nullaway/src/main/java/com/uber/nullaway/handlers/LibraryModelsHandler.java

```diff
@@ -826,7 +832,12 @@ private static class DefaultLibraryModels implements LibraryModels {
                     "getDrawable(android.content.Context,int)"))
             .add(methodRef("android.support.design.widget.TextInputLayout", "getEditText()"))
             .build();
-
+    private static final ImmutableSetMultimap<String, Integer> NULLABLE_VARIABLE_TYPE_UPPER_BOUNDS =
+        new ImmutableSetMultimap.Builder<String, Integer>()
+            .put("java.util.function.Function", 0)
+            .put("java.util.function.Function", 1)
+            .put("java.util.Map", 1)
```

Comment:
I don't recall wanting to model `java.util.Map` in this PR.  Why did we add this change?

Reply:
Sorry, I had added it for a positive test case where only one of the indices has the upper bound and we still fail for the other. But in hindsight it is not the correct way to add a model for it. I'll remove it.